markdown/usage/configuration.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Nextflow configuration
+subtitle: How configure Nextflow to work on your system
+---
+
+One of the strongest features of Nextflow is that it can run on virtually any computational infrastructure.
+It has built-in support for HPC execution schedulers such as Slurm, SGE, PBS, LSF and more as well as cloud compute infrastructure such as AWS Batch and Google Cloud.
+
+In order to get nf-core pipelines to run properly on your system, you will need to configure Nextflow so that it knows how best to run your analysis jobs.
+Each nf-core pipeline comes with "sensible defaults" for every configuration option. These are then overwritten as needed.
+
+Pipelines have configuration "profiles" that can be enabled with the command line flag `-profile`. Multiple profiles can be specified in a comma-separated list.
+Alternatively, you can create your own configuration files and supply these to the pipeline when running.
+
+There are three main types of pipeline configuration that you can use:
+
+1. [Basic pipeline configuration profiles](#basic-configuration-profiles)
+2. [Shared _nf-core/configs_ configuration profiles](#shared-nf-core-configs)
+3. [Custom configuration files](#custom-configuration-files)
+
+Note that most users will need to specify an "executor" - telling Nextflow how to subject jobs to a job scheduler (_e.g._ SGE, LSF, SLURM, PBS, AWS Batch _etc._).
```

Comment:
```suggestion
Note that most users will need to specify an "executor" - telling Nextflow how to submit jobs to a job scheduler (_e.g._ SGE, LSF, SLURM, PBS, AWS Batch _etc._).
```
I prefer submit over subject, but that's personnal, not sure what is the best way.
Feel free to dismiss

Reply:
Subject is a typo 😓 Submit is definitely the correct word 👍